certified-connectors/GMO Sign/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+	"properties": {
+	  "connectionParameters": {
+		"username": {
+		  "type": "securestring",
+		  "uiDefinition": {
+			"displayName": "顧客ID",
+			"description": "この API の 顧客ID",
+			"tooltip": "顧客ID を入力してください",
+			"constraints": {
+			  "tabIndex": 2,
+			  "clearText": true,
+			  "required": "true"
+			}
+		  }
+		},
+		"password": {
+		  "type": "securestring",
+		  "uiDefinition": {
+			"displayName": "シークレットキー",
+			"description": "この API の シークレットキー",
+			"tooltip": "シークレットキー を入力してください",
+			"constraints": {
+			  "tabIndex": 3,
+			  "clearText": false,
+			  "required": "true"
+			}
+		  }
+		}
+	  },
+	  "iconBrandColor": "#005bac",
+	  "capabilities": [],
+	  "publisher": "GMO GlobalSign Holdings K.K."

```

Comment:
@GMO-GSHD-APIGW stackOwner property is missing. Please add it. You can refer to [apiProperties template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json).